base_external_dbsource_importer/models/external_mixin.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from collections import defaultdict
+
+from psycopg2.extensions import AsIs
+
+from odoo import api, models
+
+# special columns automatically created by the ORM
+LOG_ACCESS_COLUMNS = ["create_uid", "create_date", "write_uid", "write_date"]
+MAGIC_COLUMNS = ["id"] + LOG_ACCESS_COLUMNS
```

Comment:
If you prefer, you can use the same from odoo core https://github.com/odoo/odoo/blob/3d03864e3f7c3cb7a7da26cafa1b3563dc73ff92/odoo/models.py#L334